cli/cmd.go

```diff
@@ -122,6 +122,7 @@ var Commands = []*cli.Command{
 	sendCmd,
 	stateCmd,
 	syncCmd,
+	unregisterMinerCmd,
```

Comment:
(a bit awkward to have it top-level, but I'm not sure what would be a better place)

Reply:
Yeah, wasnt sure where to put it...